modules/op-release-notes-1-4.adoc

```diff
@@ -0,0 +1,154 @@
+// Module included in the following assembly:
+//
+// * cicd/pipelines/op-release-notes.adoc
+
+[id="op-release-notes-1-4_{context}"]
+= Release notes for {pipelines-title} General Availability 1.4
+
+
+[id="new-features-1-4_{context}"]
+== New features
+{pipelines-title} General Availability (GA) 1.4 is now available on {product-title} 4.7.
+
+[NOTE]
+====
+In addition to the stable/preview Operator channel for OCP 4.7, the general availability of Pipelines 1.4 comes with the deprecated channels corresponding to OCP 4.6 and OCP 4.5. However, support for the deprecated channels will be removed in the future releases.  
+====
+
+In addition to the fixes and stability improvements, the following sections highlight what is new in {pipelines-title} 1.4.
+
+* The custom tasks have the following enhancements:
+** Pipeline results can now refer to results produced by custom tasks.
+** Custom tasks can now use workspaces, service accounts, and pod templates to build more complex custom tasks.
+
+* The `finally` task has the following enhancements:
+** The `when` expressions are supported in `finally` tasks, which provides efficient guarded execution and improved reusability of tasks.
+** A `finally` task can be configured to consume the results of any task within the same pipeline.
++
+[NOTE]
+====
+Support for `when` expressions and `finally` tasks are unavailable in the GUI or web interface of OpenShift Pipelines 4.7.
+====
+
+* Support for multiple secrets of the type `dockercfg` or `dockerconfigjson` is added for authentication at runtime.
+
+* Functionality to support sparse-checkout with the `git-clone` task is added. This enables you to clone only a subset of the repository as your local copy, and helps you to restrict the size of the cloned repositories.
+
+* You can create pipeline runs in a pending state without actually starting them. In clusters that are under heavy load, this allows Operators to have control over the start time of the pipeline runs.
+
+* Ensure that you set the `SYSTEM_NAMESPACE` environment variable manually for the controller; this was previously set by default.
+
+* A non-root user is now added to the build-base image of pipelines so that `git-init` can clone repositories as a non-root user.
```

Comment:
Do we need this cc @vdemeester I think this is upstream specific PR `https://github.com/tektoncd/pipeline/pull/3727/`

Reply:
I think it make sense to have it yes.